### PR TITLE
Corrected CPHASE

### PR DESCRIPTION
**Pull request by SimonSekavcnik**
Corrected the CPHASE gate

### CODE REVIEW

File: eqsn/gates.py [hunk 1]
```diff
@@ -247,7 +247,7 @@ def cphase_gate(self, applied_to_id, controlled_by_id):
             applied_to_id (String): Id of the Qubit on which the X gate is applied.
```
Comment (kareem1925) [inline eqsn/gates.py]: I think you mean "the Z gate is applied"

**Review by SimonSekavcnik — COMMENTED** [review]
https://qiskit.org/documentation/stubs/qiskit.circuit.library.ZGate.html

File: eqsn/gates.py (hunk 1 above)
Comment (SimonSekavcnik) [reply eqsn/gates.py]: I just changed the matrix

Comment (SimonSekavcnik) [reply eqsn/gates.py]: you are correct

**Review by kareem1925 — APPROVED** [review]

**Pull request merged**